api/bots_api/bot_lib.py

```diff
@@ -105,6 +111,67 @@ def get_config_info(self, bot_name, section=None):
         config.readfp(open(conf_file_path))  # type: ignore
         return dict(config.items(section))
 
+class EmbeddedBotHandler(object):
+    def __init__(self, client):
+        # type: (Client) -> None
+        # Only expose a subset of our Client's functionality
+        user_profile = client.get_profile()
+        self._rate_limit = RateLimit(20, 5)
+        self._client = client
+        try:
+            self.full_name = user_profile['full_name']
+            self.email = user_profile['email']
+        except KeyError:
+            logging.error('Cannot fetch user profile, make sure you have set'
+                          ' up the zuliprc file correctly.')
+            sys.exit(1)
+
+    def embedded_send_message(self, message):
```

Comment:
Same comment as above—should this be `send_message` instead of `embedded_send_message` so that the `BotHandler` implementations can have a consistent interface?